x-pack/plugins/canvas/.storybook/redux_context.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC } from 'react';
```

Comment:
I prefer `FunctionComponent` vs `FC` but I ain't going to fight about it or anything.  But, we should reach an agreement on one and stick to that.  Maybe add a lint rule to force us to one or the other.